packages/web-components/package.json

```diff
@@ -176,8 +176,7 @@
     "./dist/esm/toggle-button/define.js"
   ],
   "scripts": {
-    "tsc": "tsc",
```

Comment:
tsc and api-extractor aliases were necessery in the past when wc-3 used newer versions, thus to call the proper binaries yarn v1 needed these to be aliased. this is no longer the case